ui/imports/utils/Global.qml

```diff
@@ -92,7 +92,7 @@ QtObject {
 
     signal privacyPolicyRequested()
 
-    signal openPaymentRequestModalRequested()
+    signal openPaymentRequestModalRequested(var inputAreaModule)
```

Comment:
Ugh, that's nasty... passing the NIM backend module around with a signal :)

Reply:
The alternative now would be to register a callback in the signal as we have for other popups. Then we can connect to `onAccepted`.